aspnet/fundamentals/servers.rst

```diff
@@ -167,12 +167,12 @@ Kestrel is a cross-platform web server based on `libuv <https://github.com/libuv
 
 Learn more about working with Kestrel to create :doc:`/tutorials/your-first-mac-aspnet`.
 
+.. note:: Kestrel is not hardened for internet facing traffic and should always be used behind a secure proxy server (ex. IIS, Nginx).
```

Comment:
we don't use ex. replace with For Example

How about:
Kestrel on it's own, has vulnerabilities as an internet web server.  To secure Kestrel, it must be used behind a secure proxy server such as IIS or Nginx.